locales/cy.json

```diff
@@ -20,6 +20,12 @@
     "internal": "Mae problem gyda’r cyfleuster taliadau. Rhowch gynnig arall arni wedyn",
     "contactService": "Mae problem gyda’r cyfleuster taliadau. Rhowch gynnig arall arni wedyn"
   },
+  "paymentLinkError": {
+    "title": "Digwyddodd gwall:",
+    "invalidReference": "Rhif mae’n rhaid i fod yn 255 nod neu lai. Ni allwch ddefnyddio unrhyw rai o’r nodau canlynol < > ; : ` ( ) \" ' = &#124; \",\" ~ [ ] .",
```

Comment:
I think `&#124;` should be `|` here.

Reply:
Also, this has a dot at the end but the English version does not.

Reply:
Fixes applied as suggested.